lib/dbMapping.js

```diff
@@ -13,8 +13,7 @@ export const TYPE_OF_BUSINESS = [
   'Ofsted registered nurseries (not within a domestic premises)'
 ];
 
-export const RETEABLE_LIMIT_ANSWER = ['Yes', 'No', 'Not Applicable'];
-export const VALID_RETEABLE_LIMIT_ANSWER = ['Yes', 'Not Applicable'];
```

Comment:
@sirLisko I took this out because I couldn't find it being used anywhere. If there's a reason we should not remove it, please let me know.

Reply:
Good catch 👍 It was used for the question that was dropped.